L/Libxc/Libxc_GPU/build_tarballs.jl

```diff
@@ -5,61 +5,53 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "Libxc_GPU"
-version = v"6.1.0"
+version = v"7.0.0"
 include("../sources.jl")
 
-sources = [
-    sources;
-    DirectorySource("./bundled")
-]
-
 # Bash recipe for building GPU version
 # Notes:
 #   - 3rd and 4th derivatives (KXC, LXC) not built since gives a binary size of ~200MB
 script = raw"""
 cd $WORKSPACE/srcdir/libxc-*/
 
-# Needed for Libxc 6.1.0 as these backport some fixes on libxc master
-# On Libxc > 6.1.0 we can also remove the -DBUILD_TESTING=OFF
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/cmake-cuda.patch
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/source-fixes.patch
+ln -s $prefix/cuda/lib $prefix/cuda/lib64
 
 mkdir libxc_build
 cd libxc_build
-cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}\
     -DCMAKE_BUILD_TYPE=Release -DENABLE_XHOST=OFF -DBUILD_SHARED_LIBS=ON \
-    -DENABLE_CUDA=ON -DCMAKE_CUDA_COMPILER=$prefix/cuda/bin/nvcc -DBUILD_TESTING=OFF \
-    -DENABLE_FORTRAN=OFF -DDISABLE_KXC=ON ..
+    -DENABLE_CUDA=ON -DCMAKE_CUDA_COMPILER=$prefix/cuda/bin/nvcc \
+    -DBUILD_TESTING=OFF -DENABLE_FORTRAN=OFF \
+    -DDISABLE_KXC=ON ..
```

Comment:
Breaking lists into separate parts and sorting them is good practice (helps shorten future diffs etc.):
```suggestion
    -DBUILD_TESTING=OFF \
    -DCMAKE_CUDA_COMPILER=$prefix/cuda/bin/nvcc \
    -DENABLE_CUDA=ON \
    -DENABLE_FORTRAN=OFF \
    -DDISABLE_KXC=ON ..
```
This should of course be extended for the entire cmake "configure" call, but GitHub prevents making a suggestion for the entire cmake configure call (due to some deleted lines, apparently).